src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngProvider.cs

```diff
@@ -86,6 +86,14 @@ public override string ToString()
         // Well known NCrypt KSPs
         //
 
+        public static CngProvider MicrosoftPlatformCryptoProvider
```

Comment:
I think there should be XML docs on new APIs. Perhaps something like this to use the other properties as a template:

```suggestion
        /// <summary>
        /// Gets a <see cref="CngProvider" /> object that specifies the Microsoft Platform Crypto Storage Provider.
        /// </summary>
        /// <value>An object that specifies the Microsoft Platform Crypto Storage Provider.</value>
        public static CngProvider MicrosoftPlatformCryptoProvider
```

@carlossanlop I noticed this PR didn't get tagged "new-api-needs-documentation". Did something change around this?

Reply:
I think the bot only adds the label when there is a modified ref file on PR creation. If a ref file is added later, the bot does not add the label. Is this correct, @maryamariyan ?